halibot/halibot.py

```diff
@@ -28,6 +29,9 @@ def __init__(self, **kwargs):
 		self.log = logging.getLogger(self.__class__.__name__)
 
 		self.use_config = kwargs.get("use_config", True)
+		self.use_auth = kwargs.get("use_auth", True)
+
+		self.auth = HalAuth()
```

Comment:
Should the value of `use_auth` set the value of `auth.enabled`? It seems a little redundant to have bolth.